test/lib/mayaUsd/fileio/CMakeLists.txt

```diff
@@ -85,4 +84,25 @@ if(CMAKE_UFE_V3_FEATURES_AVAILABLE)
     )
 endif()
 
+if(CMAKE_UFE_V3_FEATURES_AVAILABLE)
+    set(TEST_DISCARD_EDITS testDiscardEdits.py)
+    mayaUsd_get_unittest_target(target ${TEST_DISCARD_EDITS})
+    if(UFE_TRIE_NODE_HAS_CHILDREN_COMPONENTS_ACCESSOR)
```

Comment:
Enable test for orphaned node discard edits on ancestor unload only if UFE support is present.